src/context/WalletProvider.tsx

```diff
@@ -572,6 +651,7 @@ const WalletProvider = function WalletProvider({ children }: { children: React.R
 
     updateWalletState(activeWalletNameRef.current, {
       isConnected: false,
+      isDisconnectable: false,
```

Comment:
Don't really understand the sense of this flag. Can you be so kind to explain it?

Reply:
This flag is needed to remove the disconnect button from the UI; please see the frontend PR

Reply:
Finally removed